ceph.spec.in

```diff
@@ -269,6 +269,9 @@ BuildRequires:	python%{_python_buildid}-prettytable
 BuildRequires:	python%{_python_buildid}-sphinx
 BuildRequires:	lz4-devel >= 1.7
 %endif
+%if 0%{?fedora} == 28
+BuildRequires:  python2-Cython
```

Comment:
i think we should add this next to 
```
BuildRequires:	python3-Cython
```
7 lines above. and instead of guarding it with `0%{?fedora} == 28`. i think, we should put
```
%if 0%{?rhel} == 7
BuildRequires:	python34-devel
BuildRequires:	python34-setuptools
BuildRequires:	python34-Cython
%if 0%{with python2}
BuildRequires:	Cython
%endif
%else
BuildRequires:	python3-devel
BuildRequires:	python3-setuptools
BuildRequires:	python3-Cython
%if 0%{with python2}
BuildRequires:	python2-Cython
%endif
%endif

```

the reason why you only spot this on f28 is, i guess, we make python3 only build on fedora>=29 by default. but python2 python binding packages can always be generated manually with
```
rpmbuild --with python2
```

Reply:
@badone since you are at here, could you take this opportunity to guard `BuildRequires:	Cython` with `%if 0%{with python2}` in `0%{?rhel} == 7` as suggested above?

also the title of the commit message in not in sync with the change.

Reply:
I think we can simplify this even further. The `Cython` package in rhel 7 provides `python2-Cython` so we can use that to install `Cython` in rhel 7, too.